src/scripts/docs/FormApp.jsx

```diff
@@ -182,6 +183,7 @@ const FormApp = () => (
 
     <section className="site-section">
       <h3 className="site-subheadline">Input</h3>
+      <p className="site-copy">We are using the <a href="http://nosir.github.io/cleave.js/">cleave</a> package to enhance out inputs with formatting/masking ability.</p>
```

Comment:
Spelling: "package to enhance **out** inputs"

Reply:
🤦‍♂️